### PR TITLE
 resource/alicloud_ess_scaling_group: new parameter launch_template_version

### PR DESCRIPTION
**Pull request by TalentNing**
TF_ACC=1 go te
st ./alicloud -v -run=TestAccAlicloudEssScalingGroup_withLaunchTemplateId -timeout=300m
go: downloading github.com/denverdino/aliyungo v0.0.0-20220218034049-d2eb7f483ee8
=== RUN   TestAccAlicloudEssScalingGroup_withLaunchTemplateId
--- PASS: TestAccAlicloudEssScalingGroup_withLaunchTemplateId (55.38s)
PASS
ok      github.com/aliyun/terraform-provider-alicloud/alicloud  57.324s

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## 1.158.0 (Unreleased)
-
+- resource/alicloud_ess_scaling_group: Adds new parameter launch_template_version
```
Comment (xinmu0426) [inline CHANGELOG.md:2]: 两次commit 合并为一个

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 - **New Resource:** `alicloud_ess_alb_server_group_attachment`
```
Comment (xinmu0426) [inline CHANGELOG.md:4]: ci没过

File: alicloud/resource_alicloud_ess_scalinggroup.go [hunk 3]
```diff
@@ -122,6 +122,10 @@ func resourceAlicloudEssScalingGroup() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"launch_template_version": {
+				Type:     schema.TypeString,
+				Optional: true,
```
[the remark this answers is not in the document]
Comment (TalentNing) [reply alicloud/resource_alicloud_ess_scalinggroup.go:127]: 没有固定的配置

File: CHANGELOG.md (hunk 1 above)
Comment (TalentNing) [reply CHANGELOG.md:3]: done

**Pull request merged**